packages/shorebird_cli/lib/src/shorebird_build_mixin.dart

```diff
@@ -138,7 +138,7 @@ mixin ShorebirdBuildMixin on ShorebirdCommand {
   Future<void> buildApk({
     String? flavor,
     String? target,
-    bool? splitPerAbi,
+    bool splitPerAbi = false,
```

Comment:
Is this enabled, or disabled by default?

Reply:
This is disabled by default.